src/community/web-resource/src/main/resources/org/geoserver/web/resources/styleResourceBrowser.css

```diff
@@ -0,0 +1,115 @@
+#page {
+	padding-bottom: 10px;
+}
+
+.treeview {	
+    height: calc(100vh - 325px);
+    width: 100%;
+	overflow-y: scroll;
```

Comment:
@NielsCharlier you fixed most tabs but sneaked in a few more. :-)

I will fix before merging as I have to deal the pom conflicts.